tests/test_data/test_pipelines/test_indoor_sample.py

```diff
@@ -115,3 +115,40 @@ def test_indoor_seg_sample():
                         'use_normalized_coord=True, ' \
                         'num_try=10)'
     assert repr_str == expected_repr_str
+
+    # test on S3DIS dataset
+    np.random.seed(0)
+    s3dis_patch_sample_points = IndoorPatchPointSample(5, 1.0, 1.0, None, True)
+    s3dis_results = dict()
+    s3dis_points = np.fromfile(
+        './tests/data/s3dis/points/Area_1_office_2.bin',
+        dtype=np.float32).reshape((-1, 6))
+    s3dis_results['points'] = DepthPoints(
+        s3dis_points, points_dim=6, attribute_dims=dict(color=[3, 4, 5]))
+
+    s3dis_pts_semantic_mask = np.fromfile(
+        './tests/data/s3dis/semantic_mask/Area_1_office_2.bin', dtype=np.long)
+    s3dis_results['pts_semantic_mask'] = s3dis_pts_semantic_mask
+
+    s3dis_results = s3dis_patch_sample_points(s3dis_results)
+    s3dis_points_result = s3dis_results['points']
+    s3dis_semantic_labels_result = s3dis_results['pts_semantic_mask']
+
+    # manually constructed sampled points
+    s3dis_choices = np.array([87, 37, 60, 18, 31])
+    s3dis_center = np.array([2.691, 2.231, 3.172])
+    s3dis_center[2] = 0.0
+    s3dis_coord_max = np.amax(s3dis_points[:, :3], axis=0)
+    s3dis_input_points = np.concatenate([
+        s3dis_points[s3dis_choices, :3] - s3dis_center,
+        s3dis_points[s3dis_choices,
+                     3:], s3dis_points[s3dis_choices, :3] / s3dis_coord_max
+    ],
+                                        axis=1)
```

Comment:
This format seems so strange. Can we move the axis=1 to the above line? Or we manually reformat it and then make it pass CI/pre-commit again.